corehq/ex-submodules/pillowtop/utils.py

```diff
@@ -354,7 +354,7 @@ def bulk_fetch_changes_docs(changes, domain=None):
 
 def get_errors_with_ids(es_action_errors):
     return [
-        (item['_id'], item['error'])
+        (item['_id'], item.get('error'))
         for op_type, item in _changes_to_list(es_action_errors)
```

Comment:
```suggestion
        for op_type, item in _changes_to_list(es_action_errors)
        if not (item.get('op_type') == 'delete' and item.get('status') == 404)
```
Looking at the sentry events I think we should filter out items where `op_type = 'delete'` and `status = 404`. There may be other cases where `error` is missing but I'd rather be specific in what we ignore. I guess another option would be to just ignore anything that doesn't have an `error` field which would probably be more robust to other 'non-error' situations.